src/org/zaproxy/zap/extension/alertFilters/ContextAlertFilterPanel.java

```diff
@@ -103,11 +103,9 @@ public AlertFiltersMultipleOptionsPanel(ExtensionAlertFilters extension, AlertFi
 			super(model);
 			this.extension = extension;
 
-			getTable().getColumnExt(0).setPreferredWidth(20);
-			getTable().getColumnExt(1).setPreferredWidth(50);
-			getTable().getColumnExt(2).setMaxWidth(30);
-			getTable().getColumnExt(3).setPreferredWidth(20);
+			getTable().getColumnExt(0).setMaxWidth(70);
```

Comment:
Well the issue I encountered was that I couldn't discover a way to disable resizing for a single column. Tried [`setResizable(false);`](http://docs.oracle.com/javase/1.5.0/docs/api/javax/swing/table/TableColumn.html?is-external=true#setResizable%28boolean%29) but it didn't work. 

The usability issue I encountered was that if you resize the dialog all columns resize, which is pointless for the Enabled column.

I'll see if I can come up with an alternative tonight.